### PR TITLE
#164069171 create empty react project

### PR DESCRIPTION
**Pull request by wasibani-roy**
**What does this PR do**
This PR creates an empty react project

**Description of tasks to be completed**
- Ensure that we create an empty react application

**How can this be tested??**

- Clone the project

   `git clone https://github.com/andela/ah-aquaman-frontend.git`

- Navigate to the project directory

   `cd ah-aquaman-frontend`

-  Install dependencies using `npm install`
- Run Application locally using `npm start`

**Any background context you want to provide**
N/A

**What are the relevant pivotal tracker stories**

- [#164069171](https://www.pivotaltracker.com/story/show/164069171)

### CODE REVIEW

**Review by CryceTruly — CHANGES_REQUESTED** [review]
Thanks, @Roy Wasibani for the great work, However first add nodemodules to .gitignore as `npm install ` will create the folder for everyone else.

**Review by CryceTruly — APPROVED** [review]

**Review by Peace-Apple — APPROVED** [review]
Good work @wasibani-roy

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"]
+}
```
Comment (AmosWels) [inline .babelrc]: Add a new line to this file

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+h1 {
+    color: #181a1b;
+    text-align: center;
+}
```
Comment (AmosWels) [inline src/App.css]: Same here, add a new line

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    render() {
+        return (
+            <div>
+                <h1>Authors Haven Application!</h1>
+            </div>
+        );
+    }
+}
+
+export default App;
```
Comment (AmosWels) [inline src/App.js]: Same here, add a new line

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <title>Authors Haven Application</title>
+</head>
+
+<body>
+    <div id="root">
+
+    </div>
+</body>
+
+</html>
```
Comment (AmosWels) [inline src/index.html]: Add a new line here

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+
+ReactDOM.render(<App />, document.getElementById("root"));
```
Comment (AmosWels) [inline src/index.js]: Add a new line here

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        use: ["style-loader", "css-loader"]
+      }
+    ]
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: "./src/index.html"
+    })
+  ]
+};
```
Comment (AmosWels) [inline webpack.config.js]: Add a new line here

File: .babelrc (hunk 1 above)
Comment (joelethan) [inline .babelrc]: @wasibani-roy Most of the files do not have an extra line at the end of the code, could you please add them

**Review by AmosWels — CHANGES_REQUESTED** [review]
Goodwork @wasibani-roy, please consider amending your commit message `footer` to follow the team conventions as agreed here https://github.com/andela/engineering-playbook/tree/test-branch/5.%20Developing/Conventions#message-footer.

- For `chores`: use `Finishes`, for `Bugs`: use `Fixes` and for `features`: use `Delivers`

- Also consider creating an `assets` directory in the `src` folder where all the css will be housed. Then move `App.css` file there and update the path in the `App.js`

**Review by Peace-Apple — APPROVED** [review]
Thanks for implementing the feedback

**Review by joelethan — APPROVED** [review]

**Review by CryceTruly — APPROVED** [review]

**Review by AmosWels — CHANGES_REQUESTED** [review]
Goodwork @wasibani-roy, please check how you named your `assets` folder. You named it from `assests`. Also check that your PR message is well formatted. See the screenshot below.

image

**Review by CryceTruly — APPROVED** [review]

**Review by CryceTruly — CHANGES_REQUESTED** [review]
Thanks for the Good work @wasibani-roy, However, the story also consists of configuring a linting library like eslint using the Airbnb option, Let's consider adding it.

**Comment by joelethan** [comment]
@CryceTruly We do not need the implementation of Airbnb. It was only to be implemented if we used create-react-app, however, we set the whole app from scratch, so this implementation is not needed.

**Review by BagzieGracious — APPROVED** [review]
Thank you @wasibani-roy for the add work

**Review by joelethan — APPROVED** [review]

**Review by CryceTruly — APPROVED** [review]

**Review by Peace-Apple — APPROVED** [review]
Good work @wasibani-roy, thanks for adding eslint

**Review by AmosWels — APPROVED** [review]

**Pull request merged**